packages/mobile/src/store/audio/reducer.ts

```diff
@@ -108,13 +108,16 @@ const reducer = (
       // Side-Effect (yes, this shouldn't be in a reducer, but
       // maybe not worth including sagas/thunks)
       if (playing) {
-        track(
-          make({
-            eventName: EventNames.PLAYBACK_PLAY,
-            id: `${state.queue[newIndex].trackId}`,
-            source: PlaybackSource.PASSIVE
-          })
-        )
+        const newTrack = state.queue[newIndex]
```

Comment:
just to i understand, this is probably when the user hits next on the end of a lineup?

Reply:
ohh that's smart, i was struggling to determine what was happening here, anything i should do different here given that context?

Reply:
i think this is ok? we should test on web a decent bit just in case